com.unity.multiplayer.mlapi/Runtime/Messaging/MessageQueue/MessageQueueContainer.cs

```diff
@@ -129,7 +127,7 @@ public uint GetStreamBufferFrameCount(MessageQueueHistoryFrame.QueueFrameType qu
                 return null;
             }
 
-            writer = BeginAddQueueItemToFrame(messageType, Time.realtimeSinceStartup, transportChannel, 0,
+            writer = BeginAddQueueItemToFrame(messageType, Time.realtimeSinceStartup, transportChannel, NetworkManager.LocalClientId,
```

Comment:
This change only would impact non-batched messages, but currently with the above change non-batched messages are only not working with loopback (Host) messages.  This assures that if non-batched messaging is fixed prior to Zero-Alloc messaging that at least the sender will always be from the localClientId and not always from the server.